qiskit_ibm_runtime/execution_span.py

```diff
@@ -1,283 +0,0 @@
-# This code is part of Qiskit.
```

Comment:
this deleted file contains three classes. they were copy-pasted as-is into the new folder files